Rakefile

```diff
@@ -5,4 +5,4 @@ require File.expand_path("config/application", __dir__)
 
 Rails.application.load_tasks
 
-task default: [:spec]
+task default: %i[spec lint]
```

Comment:
Same comment about [the behaviour of `spec`](https://github.com/alphagov/content-publisher/blob/master/Rakefile#L7) here.

Are we trying to keep the order of `lint` vs. `spec` consistent? I notice it's different between this and the previous PR I looked at.

Reply:
I think we have bigger fish to fry.

Reply:
```suggestion
Rake::Task[:default].clear
task default: %i[lint spec]
```

Reply:
But I switched the ordering anyway. 😇